platform-visualization/src/viewer.js

```diff
@@ -283,35 +306,53 @@ function onElementClick(id) {
         window.camera.disable();   
     }
 
-    function showDeveloper(id) {
+    function showDeveloper(id, _element) {
 
         var relatedTasks = [];
         
-        var image = table[id].picture;
+        //var image = _element.picture;
 
-        var section = 0;
+        var section = 0, count = 0;
         var center = objects[id].position;
         
-        for (var i = 0; i < table.length; i++) {
+        for(var x = 0; x < window.TABLE.platafrms.length; x++){
```

Comment:
Estos 3 `for`'s (tanto con el if como sin el) se repiten mucho en todo el código de varios archivos, pueden convertirse en una función. Tendrías dolores de cabeza si eso se tiene que modificar, buscando cada for.